PRE-COMMIT-README.md

```
@@ -0,0 +1,205 @@
+If you need to:
+
+* Add a new tool
+* Update hook versions
+* Modify hook behavior
+
+Make changes **only** in `.pre-commit-config.yaml` and re-run:
+
+
+pre-commit install
+pre-commit run --all-files
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)